src/js/localstorage.js

```diff
@@ -0,0 +1,187 @@
+    localStorage.setItem(positionName, resourceToSave);
+
+} 
+
+export function loadFromLocalStorage(positionName) {
+
+    const resource = localStorage.getItem(positionName);
+    const resourceToLoad = JSON.parse(resource);
+    return resourceToLoad;
+}
+
+//MAIN OBJECTS , GETTERS AND SETTERS METHODS
+
+//============================================================
+export const setGame = function () {
+const game = {
+    questions: [],
+    currentQuestion: 0,
+    path: "",
+    tier: "",
+    }
+saveToLocalStorage(game,"game");
+}
+
+export const getGame = () => {
+    return loadFromLocalStorage("game");
+};
+
+//============================================================
+export const setPlayer = function () {
+const player = {
+    name:"",
+    skipUsed: false, // useSkip przestawia na true
+    score: 0,        // increase by 1 after correct answer
+    lives: 3,        // decrease by 1 after wrong answer
+    };
+saveToLocalStorage(player,"player");
+}
+
+export const getPlayer = () => {
+    return loadFromLocalStorage("player");
+};
+
+//============================================================
+
+export const setScoreboard = () => {
+    const scoreboard = [{name: '', score: 0}];
+
+saveToLocalStorage(scoreboard, "scoreboard");
+}
+
+const getScoreboard = () => {
+    return loadFromLocalStorage("scoreboard");
+};
+
+//API PARTICULAR FUNCTIONS
+
+const setQuestions = function (...ques){ //ques == questions, could be any type
+    const game = getGame();
+    localStorage.removeItem("game"); // to prevent bugs
+    game.questions = ques;
+    saveToLocalStorage(game, "game");
+    console.log(ques);
+};
+
+
+const nextQuestion = function () {
+    const game = getGame();
+    localStorage.removeItem("game");
+    game.currentQuestion++;
+    saveToLocalStorage(game, "game");
+};
+
+const setPath = function (p) {
+   const game = getGame();
+   localStorage.removeItem("game"); 
+   game.path = p;
+   saveToLocalStorage(game, "game");
+};
+
+const setTier = function (t) {
+    const game = getGame();
+    localStorage.removeItem("game");
+    game.tier = t;
+    saveToLocalStorage(game, "game");
+};
+
+const setPlayerName = function (n) {
+    const player = getPlayer();
+    localStorage.removeItem("player");
+    player.name = n;
+    saveToLocalStorage(player, "player");
+};
+
+const useSkip = function () {
+    const player = getPlayer();
+    localStorage.removeItem("player");
+    player.skipUsed = true;
+    saveToLocalStorage(player, "player");
+};
+
+const addPoint = function () {
+    const player = getPlayer();
+    localStorage.removeItem("player");
+    player.score++;
+    saveToLocalStorage(player, "player");
+};
+
+const removeLife = function () {
+    const player = getPlayer();
+    localStorage.removeItem("player");
+    player.lives--;
+    saveToLocalStorage(player, "player");
+};
+
+const addScore = function (n, s) { //n - string, player name, s - number, score
```

Comment:
The same problem as in previous comments. `n` and `s` are terrible names for variables. Why not just call it `playerName` and `score`? After like 2 months, when you'll forgot, you will not be able te tell what arguments you have to pass to the function and nobody else will be able either.

Reply:
Done.